libs/workspace-manager/src/lib/features/workspace-manager/workspace-manager.component.scss

```diff
@@ -0,0 +1,46 @@
+@use '@angular/material' as mat;
+@use 'material/colors' as colors;
+
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.workspace-manager-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 150px;
+  background-color: mat.get-color-from-palette(colors.$primary);
+  color: mat.get-color-from-palette(colors.$primary, '300-contrast');
+  padding: 0 10px;
+
+  h1 {
+    display: flex;
+    align-items: center;
+    gap: 5px;
+    font-size: 18px;
```

Comment:
you shouldn't change font-size of h1 tag, please choose the right tag/class to fit your size